### PR TITLE
Update container images to those that support the ARM architecture

### LINKED ISSUE #317: Local build on ARM fails
The everything apps depend on mockerserver. That image is only available on amd64 platform and causes timeout when running on arm. Therefore build will fail. There is no available alternative image which does support that platform. How to deal with this? Just mention in contribution.md? Make the tests using mockserver platform dependant? Any other thoughts?

### PR DESCRIPTION
**Pull request by KyleAure**
Fixes #317
Fixes #316

### CODE REVIEW

**Comment by KyleAure** [comment]
@rieckpil any idea why Wildfly is failing here? 
https://github.com/KyleAure/microshed-testing/blob/d5d1212bbe33ada93a548f329a223edc1d33e58a/sample-apps/wildfly-app/src/test/java/org/example/app/WildFlyIT.java#L41-L47

We got a 404 at the endpoint `http://127.0.0.1:32956/myservice/app/people`
But I see in the logs that the server has started:
```txt
    00:17:44,323 INFO  [org.wildfly.extension.undertow] (ServerService Thread Pool -- 78) WFLYUT0021: Registered web context: '/myservice' for server 'default-server'
    00:17:44,373 INFO  [org.jboss.as.server] (ServerService Thread Pool -- 45) WFLYSRV0010: Deployed "myservice.war" (runtime-name : "myservice.war")
    00:17:44,402 INFO  [org.jboss.as.server] (Controller Boot Thread) WFLYSRV0212: Resuming server
    00:17:44,405 INFO  [org.jboss.as] (Controller Boot Thread) WFLYSRV0060: Http management interface listening on http://127.0.0.1:9990/management
    00:17:44,405 INFO  [org.jboss.as] (Controller Boot Thread) WFLYSRV0051: Admin console listening on http://127.0.0.1:9990
    00:17:44,408 INFO  [org.jboss.as] (Controller Boot Thread) WFLYSRV0025: WildFly Full 30.0.0.Final (WildFly Core 22.0.1.Final) started in 2317ms - Started 365 of 587 services (322 services are lazy, passive or on-demand) - Server configuration file in use: standalone.xml
```

File: sample-apps/kafka-app/build.gradle [hunk 1]
```diff
@@ -6,12 +6,16 @@ dependencies {
   providedCompile 'javax:javaee-api:8.0.1'
   providedCompile 'org.eclipse.microprofile:microprofile:3.3'
   providedCompile 'org.eclipse.microprofile.reactive.messaging:microprofile-reactive-messaging-api:1.0'
-  compile 'org.apache.kafka:kafka-clients:2.4.0'
+  
+  compile 'org.apache.kafka:kafka-clients:3.6.0'
```
Comment (appiepollo14) [inline sample-apps/kafka-app/build.gradle:10]: This update makes the integrationtest fail. Reverting to 2.4.0 solves that failure. Updating might require altering the test, please examine and solve.

Comment (KyleAure) [reply sample-apps/kafka-app/build.gradle:10]: The kafka-app integration test passed locally for me, and in the CI build: https://github.com/MicroShed/microshed-testing/actions/runs/6857446341/job/18646588128?pr=319
What failure are you seeing?

**Review by appiepollo14 — APPROVED** [review]

**Comment by appiepollo14** [comment]
@KyleAure nice work

**Pull request merged**